liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -403,7 +403,15 @@ public void checkConfigFile(File configFile, File serverDir) {
                     features.removeAll(existingFeatures);
                     if (!features.isEmpty()) {
                         log.info("Configuration features have been added");
-                        runLibertyMojoInstallFeature();
+                        Element[] featureElems = new Element[features.size() + 1];
+                        int i = 0;
+                        for (String f : features) {
```

Comment:
It's strange to increment `i` within the loop.  Since `i` is needed, it would be better to use a conventional for loop (`for (int i=0; i<features.size(); i++)`).

Reply:
Because I add the acceptLicense=true which I am debating.  If I remember correctly, adding WebSphere feature requires accepting license and not OL.

Reply:
But that is being done outside of the loop, which is fine.  It could just be `featureElems[features.size()] = element(name("acceptLicense"), "true");`

Reply:
I can change the loop style